recite/admin.py

```diff
@@ -167,8 +167,8 @@ def get_segments_dict(self, encoding, segments_file):
             try:
                 surah_number = int(surah_str)
                 ayah_number = int(ayah_str)
+                segments_dict[surah_number][ayah_number] = segments_string
```

Comment:
I like the previous way more, because you show that the exception may happen only in two lines, rather than three.
It was more clear.

Reply:
Actually the bug was here.
try: file is bismillah.mp3  -> exception occured but it was wrote to file_paths with surah_number and ayah_number of previous loop.
media file was named 002003 with real file bismillah.mp3

Reply:
Ok, think the proper way is to use try, except, **else**